packages/ckeditor5-ui/src/editorui/poweredby.ts

```diff
@@ -94,6 +95,12 @@ export default class PoweredBy extends DomEmitterMixin() {
 	private _handleEditorReady(): void {
 		const editor = this.editor;
 
+		const licenseKey = this.editor.config.get( 'licenseKey' );
+
+		if ( licenseKey && verify( licenseKey ) === 'VALID' ) {
```

Comment:
How about simplifying it down to:

```
if ( verify( editor.config.get( 'licenseKey' ) ) === 'VALID' ) {
```

?

The verify method must gracefully handle undefineds, nulls, falses etc.

Reply:
Done in 3af13cc